### PR TITLE
Merge invitation to org email

### LINKED ISSUE #4510: Merge Invited user's credentials emails
Currently we have one email per app. It should be merged into one

### LINKED ISSUE #4528: Merge Welcome emails
Currently we have 1 welcome email per app. We should have only one and add a variable app.description so each one can be unique.

### PR DESCRIPTION
**Pull request by Mussuranae**
Two things here :
- I merge the emails emails when you were invited to join an org
- I prepared a description about application in the appMailSetting object

TODO : Change the id of the template if needed

See #4510 and fix #4528 (already done but needed the description)

### CODE REVIEW

File: libs/utils/src/lib/apps.ts [hunk 1]
```diff
@@ -40,8 +41,14 @@ export const appLogo = {
...
 };
-type AppLogo = keyof typeof appLogo;
-type AppLogoValue = typeof appLogo[AppLogo];
+type AppLogoValue = typeof appLogo[App];
+
+export const appDescription = {
+  catalog: '',
+  festival: '',
+  financiers: '',
+};
```
Comment (phpgeekfr) [inline libs/utils/src/lib/apps.ts:50]: so description will be added afterwards ?

Comment (phpgeekfr) [reply libs/utils/src/lib/apps.ts:50]: If so, can you add a comment with issue number?

File: apps/backend-functions/src/templates/ids.ts [hunk 2]
```diff
@@ -12,16 +12,8 @@ export const templateIds = {
...
-        festival: 'd-ce3e57248a694cefacad49bc4c820078',
-        financiers: 'd-ce3e57248a694cefacad49bc4c820078',
-      },
-      joinOrganization: {
-        catalog: 'd-a34ce9ea59c5477f9feae8f556157b6b',
-        festival: 'd-f0c4f1b2582a4fc6ab12fcd2d7c02f5c',
-        financiers: 'd-4b8c8ebb99464f87b8eaf8223ba4b562',
-      }
+      attendEvent: 'd-ce3e57248a694cefacad49bc4c820078',
+      joinOrganization: 'd-f0c4f1b2582a4fc6ab12fcd2d7c02f5c'
```
Comment (phpgeekfr) [inline apps/backend-functions/src/templates/ids.ts:16]: remember that we will be able to remove unused templates after next release

**Review by phpgeekfr — APPROVED** [review]

File: libs/utils/src/lib/apps.ts (hunk 1 above)
Comment (Mussuranae) [reply libs/utils/src/lib/apps.ts:51]: Done, and I change the description of the pull request to not close the ticket

**Pull request merged**